gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/builder/AbstractResourceBuilder.java

```diff
@@ -373,11 +373,13 @@ protected String getHttpMethod() {
 					Number min = field.getMin();
 					Number max = field.getMax();
 
-					if (field.getType().equalsIgnoreCase("integer")) {
```

Comment:
I agree that `integer` min/max validation seems to be an untested and unused feature. But why remove it instead of fix it by adding `@Min` and `@Max` annotations?